src/LtiDeepLink.php

```diff
@@ -44,17 +44,12 @@ public function getResponseJwt($resources)
     }
 
     /**
-     * This method builds an auto-submitting HTML form to post the deep linking response message
-     * back to platform, as per LTI-DL 2.0 specification. The resulting HTML is then written to standard output,
-     * so calling this method will automatically send an HTTP response to conclude the content selection flow.
-     *
-     * @param  LtiDeepLinkResource[]  $resources The list of selected resources to be sent to the platform
-     *
-     * @todo Consider wrapping the content inside a well-formed HTML document,
-     * and returning it instead of directly writing to standard output
+     * @deprecated
```

Comment:
Once again, we shouldn't echo anything. Also... why are we building this form for people? They can't style it. If they need a form like this, they should build it themselves.